node/setup.go

```diff
@@ -73,35 +74,46 @@ func createAndStartIndexerService(
 	dbProvider DBProvider,
 	eventBus *types.EventBus,
 	logger log.Logger,
-) (*indexer.Service, indexer.TxIndexer, indexer.BlockIndexer, error) {
-
-	var (
-		txIndexer    indexer.TxIndexer
-		blockIndexer indexer.BlockIndexer
-	)
-
-	switch config.TxIndex.Indexer {
-	case "kv":
-		store, err := dbProvider(&DBContext{"tx_index", config})
-		if err != nil {
-			return nil, nil, nil, err
+) (*indexer.Service, []indexer.EventSink, error) {
+
+	eventSinks := []indexer.EventSink{}
+loop:
+	for _, db := range config.TxIndex.Indexer {
```

Comment:
Do we make sure there aren't any duplicates? If not, we should do that here, because we don't want to duplicate indexing sinks.

Reply:
There is a cleaner way to do this. Before starting the loop:

1. Lower-case all values
2. Check for dups
2a. error on dup found
3. start main loop